.evergreen/load-balancer.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/sh
+
+set -o errexit  # Exit the script with error if any of the commands fail
+
+start() {
+  echo "Starting HAProxy..."
+
+  cat <<EOF_HAPROXY_CONFIG >> $DRIVERS_TOOLS/haproxy.conf
+  defaults
+      mode tcp
+      timeout connect 5000ms
+      timeout client 5000ms
+      timeout server 5000ms
+
+  frontend mongos_frontend
+      bind *:8000
+      use_backend mongos_backend
+
+  frontend mongoses_frontend
+      bind *:8001
+      use_backend mongoses_backend
+
+  backend mongos_backend
+      mode tcp
+      server mongos 127.0.0.1:27017 check
+
+  backend mongoses_backend
+      mode tcp
+      server mongos_one 127.0.0.1:27017 check
+      server mongos_two 127.0.0.1:27018 check
+EOF_HAPROXY_CONFIG
+
+  cat $DRIVERS_TOOLS/haproxy.conf
+
+  /usr/sbin/haproxy -D -f $DRIVERS_TOOLS/haproxy.conf -p ./haproxy.pid
+
+  ADDRESS=$(curl -s http://169.254.169.254/latest/meta-data/public-hostname)
```

Comment:
What's this do?  I tried to run it  on my laptop and it hung.  Why can't we just use `localhost` as the address?

Reply:
This gets the public facing IP for the box on AWS. I thought that would be more useful to export than `localhost`.

Reply:
I'd prefer `localhost` here so that the script can be run locally as well (currently, it hangs on my laptop too). I'm not sure what benefit we get by using the public IP as DNS resolution is already tested via other test suites. It doesn't seem to add much value to this specific suite.

Reply:
I've changed this to localhost now.